pkg/apis/servicecatalog/types.go

```diff
@@ -282,6 +289,10 @@ type ServicePlan struct {
 	// AlphaServiceInstanceCredentialCreateParameterSchema is the schema for the parameters that
 	// may be supplied binding to an ServiceInstance on this plan.
 	AlphaServiceInstanceCredentialCreateParameterSchema *runtime.RawExtension
+
+	// ServiceClassRef is a reference to the service class that
+	// owns this plan.
+	ServiceClassRef v1.LocalObjectReference
```

Comment:
I think that we can use `ServiceClassName` here.  LocalObjectReference is advantageous when you anticipate needing to make a local reference into a namespaced reference in the future.  I don't think we anticipate that here.

Reply:
What is `ServiceClassName`?

Reply:
I think he means just use `string` and put the service class's name in there.  However, at some point we may need to have namespaced classes/plans at some point - so would keeping LocalObjectRef be safer?  In CF-land they have the notion of making certain services available only to certain orgs (they are not globally available to everyone), so it seems very possible we'll need that too at some point.

Reply:
On a related question, if we allow for serviceclasses to be namespaced in the future will that be a backward incompatible change for our APIs? I think its just a matter of adding a `namespace` property when needed, right?

Reply:
And... can a resource type be both namespaced and not namespaced?  Meaning, can I have one instance of ServiceClass be namespaced and nother one be un-namespaced?

Reply:
This seems the more kube-y way to do that to me. LocalObjectRef is a Name.